boards/hail/src/main.rs

```diff
@@ -467,14 +469,19 @@ pub unsafe fn reset_handler() {
     sam4l::adc::ADC0.set_client(adc);
 
     // Setup RNG
+    let etor = static_init!(
+        capsules::rng::Entropy32ToRandom<'static>,
+        capsules::rng::Entropy32ToRandom::new(&sam4l::trng::TRNG)
+    );
     let rng = static_init!(
-        capsules::rng::SimpleRng<'static, sam4l::trng::Trng>,
-        capsules::rng::SimpleRng::new(
-            &sam4l::trng::TRNG,
+        capsules::rng::RngDriver<'static>,
+        capsules::rng::RngDriver::new(
+            etor,
```

Comment:
Does `etor` stand for `entropy_to_random`? I think it is better to use the full name here for clarity!

Reply:
Fixed.